src/tests/utils/sanitize-url.spec.tsx

```diff
@@ -1,32 +1,80 @@
-import { sanitizeQueryUrl } from "../../app/utils/query-url-sanitization";
+import { containsIdentifier, isAlphaNumericString, sanitizeQueryUrl } from '../../app/utils/query-url-sanitization';
+
+describe('containsIdentifier should ', () => {
+  const list = [
+    { key: 'oIx3zN98jEmVOM', contains: true },
+    { key: 'oIx3zN98jEmVOM-4mUJzSGUANeje', contains: true },
+    { key: 'planner', contains: false }
```

Comment:
According to odata spec, `planner` could be an identifier, but we aren't in a position to support that.. Can you add more tests for other known identifier types from the actual service? Use ids from the demo tenant. Include DriveItem - files, DriveItem - folders, messageId (same form as the other Outlook based APIs), teams, anda few other workloads.

Reply:
Noted